profile/src/v100/app_preferences/p2p_links/p2p_links.rs

```diff
@@ -0,0 +1,110 @@
+use crate::prelude::*;
+
+/// Collection of clients user have connected P2P with, typically these
+/// are WebRTC connections with DApps, but might be Android or iPhone
+/// clients as well.
+pub type P2PLinks = IdentifiedVecVia<P2PLink>;
```

Comment:
IdentifiedVecVia adds Serde support and UniFFI support for all types where `T` (`Element`) is Serde and UniFFI compat...